lua/nvim-tree/marks/init.lua

```diff
@@ -1,78 +1,195 @@
+local core = {} -- circular dependency
+local lib = {} -- circular dependency
+local notify = require "nvim-tree.notify"
+local remove_file = {} -- circular dependency
+local rename_file = {} -- circular dependency
+local trash = {} -- circular dependency
 local renderer = {} -- circular dependency
+local utils = require "nvim-tree.utils"
```

Comment:
Circulars aren't good, however I imagine they will boil away once core and lib are removed.